### PR TITLE
Make events detachable

### PR DESCRIPTION
**Pull request by jboolean**

### CODE REVIEW

File: test/test-eventTarget.js [hunk 1]
```diff
@@ -41,7 +41,7 @@ describe('event target', function() {
 
   it('should have preventable events', function() {
 
-    eventTarget.on('myEvent', function(e, extraArg) {e
```
Comment (RobertWarrenGilmore) [inline test/test-eventTarget.js]: What was this?

Comment (jboolean) [inline test/test-eventTarget.js]: A stray *e*.

Comment (RobertWarrenGilmore) [inline test/test-eventTarget.js]: Huh. Okay. I guess it would still have run before that was removed.

**Comment by RobertWarrenGilmore** [comment]
Nice! I approve.

**Pull request merged**